.clang-format

```diff
@@ -0,0 +1,151 @@
+---
+Language:        Cpp
+# BasedOnStyle:  Google
+AccessModifierOffset: -1
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: true              # changed
```

Comment:
I think we disagree on this one.  I'd vote to add // clang-format off // clang-format on blocks to areas of new code where custom formatting is desirable, I'm of the opinion that the areas where we would want this are fewer than when we wouldn't, but we should probably all get a vote.  Alternatively, developers could set this as they wish, but then things could switch back and forth in the future based on who has what configs.